docs/conf/user-interface/index.md

```diff
@@ -0,0 +1,621 @@
+
+| Token Name     | Description                                                             |
+| -------------- | ----------------------------------------------------------------------- |
+| `--bui-black`  | Pure black color. This one should be the same in light and dark themes. |
+| `--bui-white`  | Pure white color. This one should be the same in light and dark themes. |
+| `--bui-gray-1` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-2` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-3` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-4` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-5` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-6` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-7` | You can use these mostly for backgrounds colors.                        |
+| `--bui-gray-8` | You can use these mostly for backgrounds colors.                        |
+
+#### Core background colors
+
+These colors are used for the background of your application. We are mostly using for now a single elevated background for panels. `--bui-bg` should mostly use as the main background color of your app.
+
+| Token Name                | Description                                      |
+| ------------------------- | ------------------------------------------------ |
+| `--bui-bg`                | The background color of your Backstage instance. |
+| `--bui-bg-surface-1`      | Use for any panels or elevated surfaces.         |
+| `--bui-bg-surface-2`      | Use for any panels or elevated surfaces.         |
+| `--bui-bg-solid`          | Used for solid background colors.                |
+| `--bui-bg-solid-hover`    | Used for solid background colors when hovered.   |
+| `--bui-bg-solid-pressed`  | Used for solid background colors when pressed.   |
+| `--bui-bg-solid-disabled` | Used for solid background colors when disabled.  |
+| `--bui-bg-tint`           | Used for tint background colors.                 |
+| `--bui-bg-tint-hover`     | Used for tint background colors when hovered.    |
+| `--bui-bg-tint-focus`     | Used for tint background colors when active.     |
+| `--bui-bg-tint-disabled`  | Used for tint background colors when disabled.   |
+| `--bui-bg-danger`         | Used to show errors information.                 |
+| `--bui-bg-warning`        | Used to show warnings information.               |
+| `--bui-bg-success`        | Used to show success information.                |
+
+#### Foreground colors
+
+Foreground colours are meant to work in pair with a background colours. Typeically this would work for icons, texts, shapes, ... Use a matching name to know what foreground color to use. These colors are prefixed with `fg` to make it easier to identify.
```

Comment:
```suggestion
Foreground colours are meant to work in pair with a background colours. Typically this would work for icons, texts, shapes, ... Use a matching name to know what foreground color to use. These colors are prefixed with `fg` to make it easier to identify.
```

Reply:
Done